edx_lint/pylint/annotations_check.py

```diff
@@ -362,15 +369,41 @@ class FeatureToggleAnnotationChecker(AnnotationBaseChecker):
             NON_BOOLEAN_DEFAULT_VALUE,
             "Feature toggle default values must be boolean",
         ),
+        ("E%d64" % BASE_ID): (
+            "missing feature toggle annotation",
+            MISSING_ANNOTATION,
+            (
+                "When a WaffleFlag/Switch object is created, a corresponding annotation must be present above in the"
+                " same module",
```

Comment:
Maybe the following change will help avoid confusion if there is an annotation, but the name doesn't match and they didn't notice that.
```suggestion
                " same module with a matching name",
```

Reply:
:+1: